openstack/nova/compute.sls

```diff
@@ -82,6 +82,10 @@ vncserver tune2:
     - require:
       - file: /etc/nova/nova.conf
 
+{% endif %}
+
+{% if not virl.mitaka and virl.cluster %}
```

Comment:
It's somewhat strange that compute.sls asks whether this is a cluster, and install.sls doesn't, but I think we can get away with having the scheduler filters constant still.